### PR TITLE
Update bg.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**

### CODE REVIEW

File: streams/bg.m3u [hunk 1]
```diff
@@ -21,21 +25,23 @@ https://cdn3.invivo.bg/codehealth_test/index.m3u8
...
+#EXTINF:-1 tvg-id="FenFolkTV.bg",Fen Folk (720p) [Not 24/7]
 https://ythlsgo.onrender.com/channel/UCptENFbwjv79hmQPymnwKcA.m3u8
+#EXTINF:-1 tvg-id="",Kanal 0
+https://old.rn-tv.com:4443/k0/stream.m3u8
 #EXTINF:-1 tvg-id="LightChannel.bg",Light Channel (480p) [Not 24/7]
 https://streamer1.streamhost.org/salive/GMIlcbgM/playlist.m3u8
 #EXTINF:-1 tvg-id="MagicTV.bg",Magic TV (720p)
 https://bss1.neterra.tv/magictv/magictv.m3u8
-#EXTINF:-1 tvg-id="PIKTV.bg",PIK TV (720p)
+#EXTINF:-1 tvg-id="PIKTV.bg",PIK TV (720p)
```
Comment (BellezaEmporium) [inline streams/bg.m3u:38]:
```suggestion
#EXTINF:-1 tvg-id="PIKTV.bg",PIK TV (720p)
```

File: streams/bg.m3u [hunk 2]
```diff
@@ -21,21 +25,23 @@ https://cdn3.invivo.bg/codehealth_test/index.m3u8
...
 #EXTINF:-1 tvg-id="EuroFolkTV.bg",EuroFolk TV (720p)
 https://eurofolk.cdn.netbadgers.com/bg040/bg041/bg041.m3u8
-#EXTINF:-1 tvg-id="",Evrokom (360p)
+#EXTINF:-1 tvg-id="Eurocom.bg",Evrokom (360p)
 https://live.ecomservice.bg/hls/stream.m3u8
-#EXTINF:-1 tvg-id="",Fen Folk (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="FenFolkTV.bg",Fen Folk (720p) [Not 24/7]
 https://ythlsgo.onrender.com/channel/UCptENFbwjv79hmQPymnwKcA.m3u8
+#EXTINF:-1 tvg-id="",Kanal 0
+https://old.rn-tv.com:4443/k0/stream.m3u8
```
Comment (BellezaEmporium) [inline streams/bg.m3u:33]: Weird SSL being used on this website. Might not work for all.

Comment (BellezaEmporium) [reply streams/bg.m3u:33]: https://old2.rn-tv.com/k0/stream.m3u8 however works on my browser.

File: streams/bg.m3u [hunk 3]
```diff
@@ -21,21 +21,23 @@ https://cdn3.invivo.bg/codehealth_test/index.m3u8
...
 #EXTINF:-1 tvg-id="EuroFolkTV.bg",EuroFolk TV (720p)
 https://eurofolk.cdn.netbadgers.com/bg040/bg041/bg041.m3u8
-#EXTINF:-1 tvg-id="",Evrokom (360p)
+#EXTINF:-1 tvg-id="Eurocom.bg",Evrokom (360p)
 https://live.ecomservice.bg/hls/stream.m3u8
-#EXTINF:-1 tvg-id="",Fen Folk (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="FenFolkTV.bg",Fen Folk (720p) [Not 24/7]
 https://ythlsgo.onrender.com/channel/UCptENFbwjv79hmQPymnwKcA.m3u8
+#EXTINF:-1 tvg-id="Kanal0.bg",Kanal 0
+https://old.rn-tv.com:4443/k0/stream.m3u8
```
Comment (freearhey) [inline streams/bg.m3u:29]:
```suggestion
#EXTVLCOPT:http-referrer=rn-tv.com
https://old2.rn-tv.com/k0/stream.m3u8
```

**Review by BellezaEmporium — DISMISSED** [review]
Referrer not needed.

File: streams/bg.m3u (hunk 3 above)
Comment (freearhey) [reply streams/bg.m3u]: Without referrer it doesn't work in VLC, at least for me.

**Review by freearhey — DISMISSED** [review]

**Comment by freearhey** [comment]
Without referrer it doesn't work in VLC, at least for me.

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**